app/controllers/application_controller.rb

```diff
@@ -421,6 +421,11 @@ def acts_like_production?
   end
   helper_method :acts_like_production?
 
+  def show_xml?
+    !(Rails.env.demo? || Rails.env.staging?)
```

Comment:
Maybe make this an allow list, like only heroku and local. We're now treating it like sensitive data so we should be specific about what envs it should be in. Production should also be excluded anyway.

Reply:
oh true whitelist is def better